06-code.Rmd

```diff
@@ -134,7 +134,8 @@ One popular example is R loops which can be particularly slow in R. Note that th
 
 - [Using lapply](https://statisticsglobe.com/avoid-for-loop-in-r)
 - [Using the purrr package](https://purrr.tidyverse.org/)
```

Comment:
maybe we should move this one down to be the last listed? I like the purrr package but beginners tend to find it challenging.

Reply:
Fair! Done.